mautic/mautic
# Normalize select values in form results email

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:

If you submit form with select and multiselect and receive form results you get values instead of labels. This PR fixed it



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create form
3. Create multiselect field with options - First => 1, Second => 2, Third => 3
4. Create form results to email action
5. Send form with two of selected options - First  and Seocnd
6. Check email - you receive values: 1, 2
7. After PR you receive in email: First, Second
